onnxruntime/python/tools/kernel_explorer/kernels/gemm_ck.cc

```diff
@@ -35,77 +36,45 @@ struct DataTypeAdaptor<half> {
   using type = ck::half_t;
 };
 
+}  // namespace
+
 using Row = ck::tensor_layout::gemm::RowMajor;
 using Col = ck::tensor_layout::gemm::ColumnMajor;
 
-}  // namespace
+using Nop = ck::tensor_operation::element_wise::PassThrough;
 
-template <typename T>
-class CKGemm : public GemmBase<T> {
+// to be moved to onnxruntime once we have a monolithicly tunable gemm wrapper and it is enabled for onnxruntime
+template <typename T, typename ALayout, typename BLayout>
+auto GetCKGemmOpTypeStringAndInstances() {
```

Comment:
There are multiple synonyms used in this file: impl, instance, op (e.g. [type_string, op]), etc, maybe just consistently use a single name, say impl?